pkg/sensors/exec/exec.go

```diff
@@ -77,9 +76,8 @@ func msgToExecveAccountUnix(m *exec.MsgExecveEventUnix) {
 	if option.Config.UsernameMetadata == int(option.USERNAME_METADATA_UNIX) {
 		if ns, err := namespace.GetMsgNamespaces(m.Unix.Msg.Namespaces); err == nil {
 			if ns.Mnt.IsHost && ns.User.IsHost {
-				// use Golang user.LookupId() as we want to only parse /etc/passwd for now
-				if userInfo, err := user.LookupId(strconv.FormatUint(uint64(m.Unix.Process.UID), 10)); err == nil {
-					m.Unix.Process.User.Name = userInfo.Name
+				if username, err := userdb.UsersCache.LookupUser(m.Unix.Process.UID); err == nil {
+					m.Unix.Process.User.Name = username
```

Comment:
would be nice to have a metric here for errors.

Reply:
Will do it separately, thanks